trino/overlays/stage/aws-secret.enc.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: stage-aws-secret
+stringData:
+    AWS_ACCESS_KEY_ID: ENC[AES256_GCM,data:sMBChgztBgq6ZpYHx5Aj0lEYxn4=,iv:06/S4cQZh72GIj6nykbVZy/QykIvD26aI3bNz2G4OWY=,tag:MHHirwX6WV7emttm+t+8rA==,type:str]
+    AWS_SECRET_ACCESS_KEY: ENC[AES256_GCM,data:P0o9054m3THwnP0Bq8fVSijWLW2f5kBvErwoUs76ifUUxWZ4yCmFYg==,iv:V0FtI2fttEa8UkYjUNCQggv5598mM2tqY+n/iy5Fsw8=,tag:Jzu1rv3rG+3BoqM+1eM2Ew==,type:str]
+    BUCKET: ENC[AES256_GCM,data:Rr5mj1yPYoFYTpyctHqVnOOWRzU9Q3E2Dbe4jOEO,iv:Zh306XIlgcKSk9XNJGaUd3JVfhhYKYJ4gbxc9C7NRL4=,tag:yQ4nlwC8u0tsahlAs2nPyw==,type:str]
```

Comment:
IMO this should default to the Playpen bucket and credentials (same for stage and dev). I think that will help us to limit access to sensitive buckets.